code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -180,10 +180,14 @@
 	if(!CI.wearer)
 		fail("Cruciform is not installed.", user, C)
 		return FALSE
+	
+	if(CI.wearer.stat != DEAD)
+		fail("This one still lives.", user, C)
+		return FALSE
 
 	var/mob/M = CI.wearer
 
-	if(ishuman(M) && M.is_dead())
```

Comment:
You can move the line 188 above 184 	if(CI.wearer.stat != DEAD) then you can use M.is_dead() replacing the check CI.wearer.stat != DEAD with this.

Reply:
What?

Reply:
`	if(CI.wearer.stat != DEAD)
		fail("This one still lives.", user, C)
		return FALSE

	var/mob/M = CI.wearer

	if(ishuman(M))`

becomes this

`	var/mob/M = CI.wearer
	
	if(!M.is_dead())
		fail("This one still lives.", user, C)
		return FALSE

	if(ishuman(M))`

The ! in front of M.is_dead() turns from FALSE to TRUE ( actually more like from 0 to 1 and vice-versa ).

Reply:
And this `var/mob/M = CI.wearer` lets us to have access to the proper mob/m and thus is possible to call functions on this without having the compiler complain because it wasn't type cased into /mob/